cvat-cli/src/cvat_cli/parser.py

```diff
@@ -47,6 +49,12 @@ def make_cmdline_parser() -> argparse.ArgumentParser:
         description="Perform common operations related to CVAT tasks.\n\n"
     )
     parser.add_argument("--version", action="version", version=VERSION)
+    parser.add_argument(
+        "--ssl-verify",
```

Comment:
at the same time, the argument name is correct. I don't know why for an env variable and a command line argument the different convention is used.